src/main/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientAdapterFactory.java

```diff
@@ -27,6 +29,7 @@
 import org.slf4j.LoggerFactory;
 
 import com.adobe.cq.commerce.graphql.client.GraphqlClient;
+import com.adobe.xfa.ut.StringUtils;
```

Comment:
Not sure if this is intentional, better use `import org.apache.commons.lang3.StringUtils;`?

Reply:
Not intentional, it's just the wrong import.